.pre-commit-config.yaml

```diff
@@ -52,6 +52,30 @@ repos:
       - id: black
         args: [--safe, --quiet]
         exclude: *fixtures
+  - repo: https://github.com/psf/black
```

Comment:
One final nit, can't this be a separate hook in the `repo` above? Or this is not how `pre-commit` works?

Reply:
Pushed a small change, let me know what you think!

Reply:
Nice !

Reply:
Same thing could be done for ruff actually

Reply:
I'll do that next time there's a pre-commit autoupgrade, (should be soon).